wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+
+Mặc dù trên hầu hết các phần cứng hiện đại, chúng tôi thấy lazer hoạt động ổn định, nhưng luôn có những trường hợp khó khăn khi mỗi người dùng có một cấu hình phần cứng khác nhau. Trong lộ trình ngắn hạn của chúng tôi, chúng tôi đang tìm cách hỗ trợ DirectX (còn gọi là "chế độ tương thích" ở chế độ ổn định) và Vulkan, cả hai đều có hỗ trợ trình điều khiển tốt hơn OpenGL trên tất cả phần cứng. Khi điều này được thực hiện, hiệu suất trên phần cứng như chipset tích hợp của Intel sẽ cải thiện rất nhiều.
+
+### Cung cấp thông tin phản hồi
+
+#### Một tính năng mà tôi phụ thuộc vào bị thiếu! / Một cái gì đó đã thay đổi và tôi không thích nó. / Tôi đã tìm thấy một lỗi, cách tốt nhất để báo cáo lỗi đó là gì?
+
+Có khả năng rất cao là chúng tôi đã biết về điều này và theo dõi nó để triển khai trong tương lai! Vui lòng tìm kiếm [trình theo dõi vấn đề](https://github.com/ppy/osu/issues) và [trang thảo luận](https://github.com/ppy/osu/discussions). Nếu bạn không thể tìm thấy bất kỳ chủ đề phù hợp nào, vui lòng [mở một cuộc thảo luận](https://github.com/ppy/osu/discussions/new).
+
+Xin lưu ý rằng chúng tôi đã theo dõi hơn 1.000 sự cố có mức độ ưu tiên khác nhau và chúng tôi có thể mất một chút thời gian để khắc phục các sự cố chỉ ảnh hưởng đến một số ít người dùng.
+
+### Khác
+
+#### Tại sao nó được gọi là "lazer"?
+
+Cái gì sắc bén hơn cuttting-edge?
+
+#### Tại sao mất quá nhiều thời gian để trở thành bản phát hành "chính"?
+
+Trong khi osu! có vẻ giống như một trò chơi đơn giản, có hàng trăm trên hàng trăm tính năng và hệ thống mà người dùng đã dựa vào. Tùy thuộc vào người bạn hỏi, lazer có thể đã ở trạng thái hoàn toàn có thể chơi được trong nhiều năm nay hoặc có thể thiếu vô số tính năng.
+
+Một lĩnh vực khác đã tốn rất nhiều công sức là bảo tồn lịch sử — đảm bảo rằng các bản đồ nhịp hoạt động chính xác như bình thường, bao gồm cả các trường hợp biên không được lên kế hoạch ban đầu. osu! là một hệ sinh thái sôi động và người dùng đã tự do mở rộng trò chơi vượt xa phạm vi dự kiến của nó và chúng tôi đang cố gắng hết sức để nắm bắt và hỗ trợ điều này trong tương lai.
+
+Cuối cùng, không giống như lần lặp lại trước, chúng tôi đang dành thời gian và sự siêng năng để đảm bảo cơ sở mã sẽ phục vụ tốt cho chúng tôi trong tương lai. Chúng tôi đã hoàn thành nền tảng để cho phép các tính năng mới xuất hiện trực tuyến với tốc độ chóng mặt trong tương lai. Điều này sẽ bao gồm các thành phần giao diện người dùng mới, các cách mới để tạo giao diện cho trò chơi, hệ thống nhiều người chơi mới và đừng quên khả năng tải và phát tất cả các bản đồ nhịp hiện có của bạn trên các chế độ trò chơi hoàn toàn mới (còn gọi là bộ quy tắc)!
+
+#### Còn gì nữa?
+
+Chúng tôi có một lượng lớn các tính năng và cải tiến do người dùng yêu cầu mà chúng tôi sẽ tiếp tục phát triển với tốc độ ánh sáng. Đối với những người đã tham gia với chúng tôi gần đây và chưa trải nghiệm động lực của osu! phát triển, chuẩn bị để được ở trong một bất ngờ.
```

Comment:
```suggestion
Chúng tôi có một lượng lớn các tính năng và cải tiến do người dùng yêu cầu mà chúng tôi sẽ tiếp tục phát triển với tốc độ ánh sáng. Đối với những người đã tham gia với chúng tôi gần đây và chưa trải nghiệm động lực của việc phát triển osu!, chuẩn bị đón chờ một bất ngờ.
```
At this point, I really want to close this PR 👎🏼

Reply:
Sorry.